Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -117,7 +155,7 @@ const SubmissionDetails = () => {
         [ currentSubmission, canAccessAdministration ],
     );
 
-    if (isNil(currentSubmission)) {
+    if (isNil(currentSubmission) || isNil(contest)) {
```

Comment:
Seems odd that the whole page wont be rendered if the contest is not loaded. Can you work around this and not render some elements maybe?

Reply:
I removed this check there and now if the contest is null, then the button to the contest is just disabled. As soon as it is loaded, the button is enabled.